solutions/usaco-920.mdx

```diff
@@ -9,6 +9,60 @@ author: Sofia Yang
 
 <LanguageSection>
 
+<CPPSection>
+	
+```cpp
```

Comment:
this is taken from the official analysis, right? maybe add a comment at the top of the code mentioning that?

Reply:
not resolved?

Reply:
Resolved, now it is mentioned with link